terraform-docs/action.yaml

```diff
@@ -50,16 +83,27 @@ runs:
             log-info "File ${README_FILE} does not exist. Terraform-docs will create new README.md file."
         fi
       continue-on-error: true
-    - id: run-terraform-docs
-      if: steps.validate-readme.outcome == 'success'
+
+    - id: generate-example-docs
+      if: steps.inject-config-files.outputs.examples-folder-exists == 'true'
+      uses: terraform-docs/gh-actions@aeae0038ed47a547e0c0fca5c059d3335f48fb25
+      with:
+        output-method: inject
+        git-push: "true"
+        recursive: true
+        recursive-path: "examples"
+        config-file: examples/.terraform-docs.yml
+
+    - id: generate-project-docs
+      if: steps.validate-root-readme.outcome == 'success'
       uses: terraform-docs/gh-actions@aeae0038ed47a547e0c0fca5c059d3335f48fb25
       with:
         working-dir: ${{ inputs.readme-file-path }}
-        output-file: README.md
         output-method: inject
```

Comment:
you do not need `inject` here, it's the default ;-)

ref. <https://terraform-docs.io/user-guide/configuration/output/>

Reply:
same as above